test/js-api/wasm-module-builder.js

```diff
@@ -851,8 +847,9 @@ class WasmModuleBuilder {
             header.emit_u8(decl.type);
           }
 
+          section.ensure_space(5 + header.length + func.body.length);
```

Comment:
might be nice to have a const instead of 5 here

Reply:
Ack, done.